docs/3.api/4.commands/module.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "nuxi module"
+description: "Module utilities for your Nuxt application"
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/cli/blob/main/src/commands/module/add.ts
+    size: xs
+---
+
+Nuxi provides a few utilities to work with [Nuxt modules](/modules) seamlessly.
+
+## nuxi module add
+
+```bash [Terminal]
+npx nuxi module add <NAME>
+```
+
+Option        | Default          | Description
+-------------------------|-----------------|------------------
+`NAME` | - | The name of the module to install.
+
+The command lets you install [Nuxt modules](/modules) to your application with no manual work.
+
+When running the command, it will:
+- install the module as dependency using your package manager
+- add it to your [package.json](/docs/guide/directory-structure/package)
```

Comment:
```suggestion
- add it to your [package.json](/docs/guide/directory-structure/package) file.
```

Reply:
Do we need to specify this (or maybe before the installation) ? It should also update the lock file (not sure if it is relevant to mention it tho)

Reply:
I think the lock file might be too much to add but I still think it's good to mention the package.json

Reply:
@huang-julien technically I suppose both are valid options, we can either specify everything it does or mention the most important things

Reply:
In chronological order it first installs before overwriting package.json, so I don't think there's a reason to mention it *before* the installation. (Although it does both do it doesn't really matter that much, as you can add packages that aren't installed to package.json, and I guess you'll just receive an error)